doc/api.apib

```diff
@@ -145,7 +148,9 @@ aglio -i doc/api.apib -o api.html --theme-template triple --theme-variables stre
 
         {
             "enabled": "true",
-            "docker_image": "busybox:latest"
+            "docker_image": "luigi-canary-dummy:dummy",
+            "commit_sha": "f043333085fa87738ac24f04d64fb58ecc845111",
+            "next_natural_trigger": null
```

Comment:
A PR addressing #116 has been created. I think we should put the desired response here, that means `next_natural_trigger` will have a value instead of `null`.